<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioPlayer</class>
 <widget class="QWidget" name="RadioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RadioPlayer</string>
  </property>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRadio">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>30</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
  </widget>
  <widget class="QPushButton" name="vUpButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Vol Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vDownButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Vol Down</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
